computer_vision/fast_corner.py

```diff
@@ -0,0 +1,34 @@
+import cv2 as cv
+
+"""
+Features from accelerated segment test (FAST) Corner Detector
+https://en.wikipedia.org/wiki/Features_from_accelerated_segment_test
+"""
+
+
+def fast_detector(img):
+    fast = cv.FastFeatureDetector_create()
+    """
+    Drawing keypoints
+    """
+    kp = fast.detect(img, None)
+    img2 = cv.drawKeypoints(img, kp, None, color=(255, 0, 0))
+    # Print all default params
+    print(f"Threshold: {fast.getThreshold()}")
+    print(f"nonmaxSuppression:{fast.getNonmaxSuppression()}")
+    print(f"neighborhood: {fast.getType()}")
+    print(f"Total Keypoints with nonmaxSuppression: {len(kp)}")
+    cv.imwrite("fast_true.png", img2)
+    """
+    NON_MAX(NMS) Supression Disabled
+    """
+    fast.setNonmaxSuppression(0)
```

Comment:
```suggestion
    fast.setNonmaxSuppression(0)  # NON_MAX(NMS) Supression Disabled
```
Similiar to above - 
Is this comment even necessary though?

Reply:
I guess not, will update. Thanks for the corrections.